controllers/controller/devworkspacerouting/sync_routes.go

```diff
@@ -55,6 +55,10 @@ func (r *DevWorkspaceRoutingReconciler) syncRoutes(routing *controllerv1alpha1.D
 		if contains, idx := listContainsRouteByName(specRoute, clusterRoutes); contains {
 			clusterRoute := clusterRoutes[idx]
 			if !cmp.Equal(specRoute, clusterRoute, routeDiffOpts) {
+				r.Log.Info("Updating route")
```

Comment:
Route name is not in log context only workspace id and probably name/namespace.
So, maybe adding object name makes sense as well.

Reply:
Added